packages/blade/src/tokens/global/breakpoints.ts

```diff
@@ -1,25 +1,69 @@
 export type Breakpoints = Readonly<{
-  /** max width: 320px  */
+  /**
+   * `base` is used for base styling. It starts from 0px till the next existing token
```

Comment:
Optional: we can make it bit more obvious right in the beginning.

```suggestion
   * `base` is used for base styling following a **mobile first** approach. It starts from 0px till the next existing token
```